tools/pot/openvino/tools/pot/algorithms/quantization/accuracy_aware_gna/algorithm.py

```diff
@@ -24,10 +26,18 @@ def __init__(self, config, engine):
         super().__init__(config, engine)
         self._config['annotation_free'] = False
         self._config['convert_to_mixed_preset'] = False
+        self._config['bias_correction_after_modify'] = self._config.get('bias_correction_after_modify', False)
+        self._config['change_only_weights'] = self._config.get(
```

Comment:
I think, at least, this parameter can be internally used based on the `device` from the configuration, not as a separate option.

Reply:
I change that
1) use 'unify_conv_activation_weight'. when true, the q-bit for conv activation and weight are unified. (for GNA3.0), when false, we optimize them separately (for GNA3.5)
2) when the device is 'GNA', we only modify the fq for weights as the same as the previous.